### ISSUE TITLE
keystore date is wrong when list by keytool

### ISSUE THREAD

**Issue by yougg** [pavel-v-chernykh/keystore-go#4]
I tried set CreationDate to `time.Now()` for PrivateKeyEntry,   
and use `cert.NotBefore` for trustedCertEntry.

`keytool -list -keystore mykeystore.jks -storepass asdfasdf`

> Keystore type: JKS  
Keystore provider: SUN  
　  
Your keystore contains 3 entries  
　  
xxx-ca, `Jan 16, 1970`, trustedCertEntry,   
Certificate fingerprint (SHA-256): DA:2A:BE:92:25:D7:95:A3:80:E2:89:9D:53:0E  
yyy-ca, `Jan 18, 1970`, trustedCertEntry,   
Certificate fingerprint (SHA-256): B8:14:1F:4A:8B:93:4A:E5:8C:01:06:49:4A:9D  
mycert, `Jan 18, 1970`, PrivateKeyEntry,   
Certificate fingerprint (SHA-256): 6C:06:2F:0D:64:C7:7C:25:48:C4:69:71:6D:85  

also i tried export the certificate from keystore, and check the date, it's correct.

```bash
keytool -exportcert -file myks.crt -alias mycert -storepass asdfasdf -keystore mykeystore.jks -rfc
openssl x509 -noout -text -in myks.crt
```
> ......  
        Validity  
            Not Before: Sep 15 17:30:21 2017 GMT  
            Not After : Sep 13 17:30:21 2027 GMT  
......

please help check the issue, thanks.

**Comment by pavel-v-chernykh** [comment]
Hi, @yougg. Thanks for the report.
Have fixed the issue in 63bed89
Please, check, does it work for you?

**Comment by yougg** [comment]
@pavel-v-chernykh :+1:  yes, it works. thanks again
